src/applications/healthcare/questionnaire/tests/e2e/01.questionnaire.demographics.cypress.spec.js

```diff
@@ -0,0 +1,71 @@
+const Timeouts = require('platform/testing/e2e/timeouts.js');
+
+import basicUser from './fixtures/users/user-basic.json';
+
+describe('healthcare questionnaire -- demographics', () => {
+  before(() => {
+    cy.server();
```

Comment:
This is set up by default now and no longer required to be explicitly invoked.

Reply:
I saw that, but I get this error, 

<img width="992" alt="Screen Shot 2020-08-31 at 4 41 02 PM" src="https://user-images.githubusercontent.com/1793923/91766604-d6dc0d00-eba8-11ea-846c-4f87ad89fc97.png">

my thought is that the `before` is running before the server is getting set up. 

I do not see where we are calling `cy.server`. Do I need to update to master to get that change?

Reply:
Only if you haven't branched off of master in a very long time. It's been called by default for a while.

I see you're calling this in `before`. `cy.server()` is actually being called on `beforeEach` in the support file (though I'm not sure it matters whether it's in `before` or `beforeEach`.

But `cy.route` and `cy.visit` should be called in `beforeEach` rather than `before` if you intend to use them across test cases.

Reply:
It also seems your test cases might be relying on previous ones, but [best practice is to isolate them more](https://docs.cypress.io/guides/references/best-practices.html#Having-tests-rely-on-the-state-of-previous-tests).

Reply:
I choose `before`  or `beforeEach` since I was getting an error and only had to do that logic once for all the tests. But I took your advice and isolated the tests a little more and got `beforeEach` to work.